crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -146,6 +146,14 @@ impl Mesh {
     pub const ATTRIBUTE_UV_0: MeshVertexAttribute =
         MeshVertexAttribute::new("Vertex_Uv", 2, VertexFormat::Float32x2);
 
+    /// Alternate texture coordinates for the vertex. Use in conjuction with
+    /// [`Mesh::insert_attribute`].
+    ///
+    /// Typically, these are used for lightmaps, textures that provide
+    /// precomputed illumination.
+    pub const ATTRIBUTE_UV_1: MeshVertexAttribute =
+        MeshVertexAttribute::new("Vertex_Uv_1", 2, VertexFormat::Float32x2);
```

Comment:
Either:
```suggestion
        MeshVertexAttribute::new("Vertex_Uv_1", 7, VertexFormat::Float32x2);
```
Or insert at 3 and move the rest up one. Though I remember seeing a PR to serialise this, including the id, iirc. It doesn't seem to be merged yet looking at the code.

Reply:
Oh, I missed that, but the rest of the PR used 3 so this should also be 3 I think.